indra/llcommon/llthread.cpp

```diff
@@ -299,6 +300,7 @@ void LLThread::start()
     {
         mThreadp = new std::thread(std::bind(&LLThread::threadRun, this));
         mNativeHandle = mThreadp->native_handle();
+        mThreadp->detach();
```

Comment:
Given that we have explicit logic to wait for thread termination, but time out and forcibly kill the thread, this makes sense, yes. In effect we manually "try to `join()`."